htdocs/core/boxes/box_graph_nb_tickets_type.php

```diff
@@ -128,6 +128,11 @@ public function loadBox($max = 5)
 			}
 			$dataseries = array();
 			$data = array();
+			$data['COM'] = 0;
+			$data['HELP'] = 0;
+			$data['ISSUE'] = 0;
+			$data['REQUEST'] = 0;
+			$data['OTHER'] = 0;
```

Comment:
Do we really have to init array with hard coded value.
Just after we make a select to fill dynamically $data to avoid to know hard coded values.

Reply:
![image](https://user-images.githubusercontent.com/3624836/111704611-b2dbd280-883f-11eb-9313-6d84d1a1374a.png)
some value may not initialised

Reply:
I think we should instead fix the line 145 by testing if key exists in array and if not setting 0